docs/section5/Rule5-41.md

```diff
@@ -0,0 +1,46 @@
+
+# Envelope - Rule 5-41
+
+**Rule ID:** 5-41
+**Rule Description:** Opaque roof surfaces that are not regulated (not part of opaque building envelope) must be modeled with the same thermal emittance and solar reflectance in the baseline as in the proposed design. 
+**Appendix G Section:** Section G3.1-5 Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**
+
+  - get_surface_conditioning_category()
+  - get_opaque_surface_type()
+  - match_data_element()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dictionary_b = get_surface_conditioning_category(B_RMR)`  
+
+- For each building segment in the Proposed model: `for building_segment_b in B_RMR.building.building_segments:`  
```

Comment:
1. This refers to the proposed model but is looping through the building segments in the baseline model.

2. The syntax we have been using is `for building_segment_b in B_RMD...building.building_segments:`  to indicate that the RDS should loop through the buildings in the RMD.